packages/react-paypal-js/src/v6/hooks/usePayLaterOneTimePaymentSession.test.tsx

```diff
@@ -0,0 +1,381 @@
+import { act, renderHook } from "@testing-library/react-hooks";
+
+import { usePayPal } from "./usePayPal";
+import { usePayLaterOneTimePaymentSession } from "./usePayLaterOneTimePaymentSession";
+import { useProxyProps } from "../utils";
+
+import type { OneTimePaymentSession } from "../types";
+
+jest.mock("./usePayPal", () => ({
+    usePayPal: jest.fn(),
+}));
+
+jest.mock("../utils", () => ({
+    useProxyProps: jest.fn(),
+}));
+
+describe("usePayLaterOneTimePaymentSession", () => {
+    beforeEach(() => {
+        // mocking this for each test rather than a module so it can be easily unmocked
+        // where needed
+        (useProxyProps as jest.Mock).mockImplementation(
+            (callbacks) => callbacks,
+        );
+    });
+
+    it("should create a pay later payment session when the hook is called", () => {
```

Comment:
I expanded this test for venmo to also check if the callbacks when called trigger the one's passed in via props: https://github.com/paypal/paypal-js/pull/702/files#diff-d8c8d43bcb5e32ddfe9cbe9b1ecfb4afadaeed0feddca7fcf6dc12423835d92dR104-R111

Is this something that would be useful here too?